### PR TITLE
Bump version, update js-binarypack with remove unused check.

### PR DESCRIPTION
**Pull request by soleilnoirmedia**
* Updated dependency js-binarypack to v 0.0.9, and modified build task accordingly.
* New js-binarypack removes unused check.

It should resolve issues related to the binary service check: console errors, lack of secure connection support, etc.
Since the manner js-binarypack is now built (i.e. their use of of Browserify) the build task for binaryjs had to be modified as well. I would propose use a similar build for Binaryjs (I would be willing to open a PR later on).

### CODE REVIEW

**Comment by bryancatanzaro** [comment]
Just wanted to add a +1 to this pull request: as a new binaryjs user, I ran into this console error and was a little concerned. Would be good to banish it.

**Comment by soleilnoirmedia** [comment]
@bryancatanzaro Thanks for the support. If you use the forked version before the pull request is approved please let me know, specially if you see any bugs. I am very interested in seeing this pull request merged and I would be glad to try to squash some bugs if they appear.

**Comment by bryancatanzaro** [comment]
@soleilnoirmedia I'm testing your fork, right now everything looks good - the only difference I notice is that the console error is gone. =)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://secure.travis-ci.org/binaryjs/binaryjs.png)](http://travis-ci.org/binaryjs/binaryjs)
+[![Build Status](https://secure.travis-ci.org/soleilnoirmedia/binaryjs.png)](http://travis-ci.org/soleilnoirmedia/binaryjs)
```
Comment (bgruening) [inline README.md]: This commit shouldn't be part of the PR I guess :)

Comment (soleilnoirmedia) [inline README.md]: @bgruening I have fixed the issue, if necessary I can squash(rebase) the commits.

**Comment by soleilnoirmedia** [comment]
Beside @ericz and or @michelle who else can approve/merge a pull request? Support for the library seems to have died for some time. It would be good if other users or orgs. would become involved....
